corelight_md.md

```diff
@@ -1,52 +1,57 @@
-# Collect Corelight sensor logs
+# Collect Corelight Sensor logs
 
-This document describes how you can collect Corelight sensor logs by configuring the Corelight sensor and a Chronicle forwarder. This document also lists the supported log types and supported Corelight versions.
+This document describes how you can collect Corelight sensor logs by configuring the Corelight sensor and a Chronicle forwarder. This document also lists the supported log types and Corelight version.
 
 For more information, see [Data ingestion to Chronicle](https://cloud.google.com/chronicle/docs/data-ingestion-flow).
 
 ## Overview
 
-The following deployment architecture diagram shows a Corelight sensor configured to send logs to Chronicle. A customer deployment can differ from this example, and might be more complex.
+The following deployment architecture diagram shows how Chronicle sensor configured to send logs to  Chronicle . Each customer deployment might differ from this representation and might be more complex.
```

Comment:
This should say "Corelight sensor", not "Chronicle sensor"

Reply:
Updated.